.github/workflows/ci.yml

```diff
@@ -36,12 +31,34 @@ jobs:
         if: steps.cache.outputs.cache-hit != 'true'
         run: npm ci
 
+  suite:
+    needs: prepare
+    name: Suite (${{ matrix.os}}, Node ${{ matrix.node }})
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        node: [14]
```

Comment:
Any thoughts on including Node.js 18 in this list? Node 18 is unique since it introduces Web APIs that could affect our libraries, including `fetch`, `Response`, and `Blob`.

Reply:
Agree! Let's do this